Cargo.toml

```diff
@@ -38,3 +39,7 @@ jsonwebkey = "0.3.5"
 semver = "1.0.27"
 mime = "0.3.17"
 thiserror = "2.0.17"
+corim-rs = { git = "https://github.com/veraison/corim-rs" }
+chrono = "0.4.42"
+cmw = { git = "https://github.com/veraison/rust-cmw" }
```

Comment:
Please use the version published at https://crates.io/crates/cmw (0.1.0) rather than the git URL.